BenchmarkTests/BenchmarkTests.xcodeproj/project.pbxproj

```diff
@@ -1197,7 +1969,7 @@
 				INFOPLIST_KEY_UILaunchStoryboardName = LaunchScreen;
 				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
 				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
-				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
```

Comment:
suggestion/ We have two `xcconfig` files set up for Benchmarks project, this setting could easily go there to simplify configuration.

Reply:
The `xcconfig` is currently applied to the `Runner` target only.
What I did instead is set the `IPHONEOS_DEPLOYMENT_TARGET` at the project level and remove overrides in the targets.